pontoon/sync/management/commands/sync_projects.py

```diff
@@ -58,16 +58,25 @@ def handle(self, *args, **options):
         sync_log = SyncLog.objects.create(start_time=timezone.now())
 
         projects = Project.objects.filter(disabled=False)
-        slugs = options['projects'].split(',') if 'projects' in options and options['projects'] else None
+        slugs = (
+            options['projects'].split(',') if 'projects' in options and options['projects']
+            else None
+        )
         if slugs:
             projects = projects.filter(slug__in=slugs)
 
         if len(projects) < 1:
             raise CommandError('No matching projects found.')
 
         if slugs and len(projects) != len(slugs):
-            invalid_slugs = sorted(set(slugs).difference(set(projects.values_list('slug', flat=True))))
-            self.stderr.write('Couldn\'t find projects with following slugs: {}'.format(', '.join(invalid_slugs)))
+            invalid_slugs = sorted(
+                set(slugs).difference(set(projects.values_list('slug', flat=True)))
+            )
+            self.stderr.write(
+                'Couldn\'t find projects with following slugs: {}'.format(
```

Comment:
Nit, unrelated to PEP8, and personal opinion: I'd prefer `"Couldn't find projects with following slugs"`.